guides/javascripts/smoothScrolling.js

```diff
@@ -0,0 +1,18 @@
+
+let anchorlinks = document.querySelectorAll('a[href^="#"]')
+ 
+for (let item of anchorlinks) { //  
+    item.addEventListener('click', (e) => {
+        let hashval = item.getAttribute('href')
+        let target = document.querySelector(hashval)
+        target.style.paddingTop = '75px'; // To compensate navigation bar height
+        target.scrollIntoView({
+            behavior: 'smooth',
+            block: 'start',
+            inline: 'nearest'
+        })
+        history.pushState(null, null, hashval)
+        e.preventDefault()
+    })
+}
```

Comment:
```suggestion
}

```
Just missing a new line at the end of the file.